lib/adal.js

```diff
@@ -1232,6 +1376,10 @@ var AuthenticationContext = (function () {
         }
     };
 
+    /**
+     * Creates a copy of the config object passed to the constructor.
```

Comment:
The description should only describe what this method does. You can write a generic description for this: returns cloned copy of the passed object, or something along those lines